pyproject.toml

```
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["setuptools>=67.7.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+
+name = "ghga_validator"
+version = "1.0.0"
+description = "GHGA Validator - A Python library and command line utility to validate metadata"
+readme = "README.md"
+authors = [
+    { name = "German Human Genome Phenome Archive (GHGA)", email = "contact@ghga.de" },
+]
+requires-python = ">=3.9"
+license = { text = "Apache 2.0" }
+classifiers = [
+    "Development Status :: 1 - Planning",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "License :: OSI Approved :: Apache Software License",
+    "Topic :: Internet :: WWW/HTTP :: HTTP Servers",
+    "Topic :: Software Development :: Libraries",
+    "Intended Audience :: Developers",
+]
+dependencies = [
+    "typer >= 0.7.0",
+    "pydantic >1.10.6, <2",
+    "linkml >= 1.5.7, <1.6.0",
+    "linkml-validator >= 0.4.5"
```

Typer should also have an upper bound. Moreover, I pinned the linkml dependencies as they are very unreliable and we can only guarantee that thinks work for a very specific version of them:

```suggestion
    "typer ~= 0.7.0",
    "pydantic >1.10.6, <2",
    "linkml == 1.5.7, <1.6.0",
    "linkml-validator == 0.4.5"
```

---

In metldata we have upgraded to the latest versions so we should use them in the tools, too
```
    "linkml==1.6.1",
    "linkml-runtime==1.6.0",
    "linkml-validator==0.4.5",
```